File: TUF/String/CountNumberOfWords.java
import java.util.Scanner;

public class CountNumberOfWords {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the string: ");
        String str = input.nextLine();
        input.close();
        System.out.println("Number of words: " + countNumberOfWords(str));
    }

    private static int countNumberOfWords(String str) {
        int count = 1;
        for (int i = 0; i < str.length(); i++) {
            if (str.charAt(i) == ' ') {
                count++;
            }
        }
        return count;
    }
}

File: TUF/String/CountNumberOfWords.ts
function countNumberOfWords(str: string): number {
    let count: number = 1;
    for(const char of str) {
        if(char == ' '){
            count++;
        }
    }
    return count;
}

let str: string = "Hello World";
console.log(countNumberOfWords(str));

export {};
